src/DocBlock/Generator.php

```diff
@@ -56,8 +56,12 @@ public function addParams(Node\Stmt\ClassMethod $node, $text)
 
         foreach ($node->getParams() as $param) {
             if ($param->type && $param->type instanceof Node\Name) {
```

Comment:
can we extract the Type generation into a seperate function like `getParamType(Param $param)` ? Just to make the function less complex